bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -10,6 +10,71 @@
 				<label>Fixed Heating Target Return Temperature</label>
 			</add-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<!-- remove renamed channels -->
+			<remove-channel id="channel232"/>
+			<remove-channel id="channel233"/>
+			<remove-channel id="channel237"/>
+			<remove-channel id="channel238"/>
+			<remove-channel id="channel239"/>
+			<remove-channel id="channel240"/>
+			<remove-channel id="channel241"/>
+			<remove-channel id="channel242"/>
+			<remove-channel id="channel243"/>
+			<remove-channel id="channel252"/>
+			<remove-channel id="channel258"/>
+			<remove-channel id="channel260"/>
+
+			<!-- add renamed channels -->
+			<add-channel id="temperatureVapourisation">
+				<type>Number:Temperature</type>
+				<label>Temperature Vapourisation</label>
```

Comment:
A bit of a nit:
Explicilty adding this type and label should have no effect. If they are not provided here, the label and type from the channel type are used. (applies to all)